Description: fix test for rollup ≥ 1.32.1
Author: Xavier Guimard <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2020-09-21

--- a/legacy/test/test.js
+++ b/legacy/test/test.js
@@ -3,7 +3,7 @@
 
 const assert = require('assert');
 const { rollup } = require('rollup');
-const replace = require('../dist/rollup-plugin-replace.cjs.js');
+const replace = require('rollup-plugin-replace');
 const fs = require('fs');
 const { SourceMapConsumer } = require('source-map');
 const { getLocator } = require('locate-character');
@@ -179,10 +179,10 @@
 			const bundle = await rollup({
 				input: 'main.js',
 				onwarn(warning) {
-					assert.equal(
-						warning.message,
-						"Sourcemap is likely to be incorrect: a plugin ('replace') was used to transform files, but didn't generate a sourcemap for the transformation. Consult the plugin documentation for help"
-					);
+					console.error(warning.message);assert.ok(
+						warning.message.match(new RegExp(
+						"^Sourcemap is likely to be incorrect: a plugin \\('?replace'?\\) was used to transform files, but didn't generate a sourcemap for the transformation. Consult the plugin documentation for help$"
+					)));
 					warned = true;
 				},
 				plugins: [
