Description: fix legacy plugin for rolup 3
Author: Yadd <yadd@debian.org>
Forwarded: not-needed
Last-Update: 2022-10-14

--- a/legacy/rollup.config.js
+++ b/legacy/rollup.config.js
@@ -1,9 +1,9 @@
-import buble from 'rollup-plugin-buble';
-import pkg from './package.json';
+const buble = require('rollup-plugin-buble');
+const pkg = require('./package.json');
 
-var external = Object.keys(pkg.dependencies).concat('path');
+const external = Object.keys(pkg.dependencies).concat('path');
 
-export default {
+module.exports = {
 	input: 'src/index.js',
 	plugins: [ buble() ],
 	external,
